packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -97,18 +97,18 @@ class AttachCommand extends FlutterCommand {
       ..addOption(
         'debug-port',
         hide: !verboseHelp,
-        help: '(deprecated) Device port where the observatory is listening. Requires '
+        help: '(deprecated) Device port where the Dart VM Service is listening. Requires '
```

Comment:
Maybe it's time to delete this flag? Or do we usually leave deprecated flags in for longer @christopherfujino?

Reply:
I'm on board with removing this flag, but I'd rather not do it as part of this PR just in case it breaks something down stream.

Reply:
Let me ask Hixie (who noted its deprecated) in discord...

Reply:
Yeah it doesn't have to be in this PR, if at all.